packages/core/scripts/copy-build-info.js

```diff
@@ -47,10 +47,48 @@ assert(
     readJSON('artifacts/@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol/ProxyAdmin.dbg.json').buildInfo,
 );
 
-const buildInfo = readJSON(jsonRelativePath);
-const reducedInfo = { solcLongVersion: buildInfo.solcLongVersion, input: buildInfo.input };
+let buildInfo = readJSON(jsonRelativePath);
 
-const sources = reducedInfo.input.sources;
+// Keep only relevant sections of output.contracts
+const contractFiles = buildInfo.output.contracts;
+for (const contractFile in contractFiles) {
+  const contractNames = contractFiles[contractFile];
+  for (const contractName in contractNames) {
+    contractNames[contractName] = {
+      abi: contractNames[contractName].abi,
+      evm: contractNames[contractName].evm,
+      metadata: contractNames[contractName].metadata, // metadata is needed to determine the license type
+    };
+  }
+}
+
+// Adjust correponding output selection
```

Comment:
This is clever... but we should test that it actually works and that the end result can be verified. I don't trust the Solidity compiler to produce exactly the same bytecode with the modified settings. Perhaps we should: 1) check with the Solidity team if they think this should work, and 2) manually check that we can verify contracts like this (you may have already tried?).

Reply:
I was able to verify a contract using this.

Would it be safer (although a bit incorrect) to keep the original output selection here, even if we remove it from the actual output?

Reply:
Yeah I'd feel more confident in that. I think that + using the `-slim` format would be good.